src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5945,6 +5945,15 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_BadSwitchValue" xml:space="preserve">
     <value>Command-line syntax error: '{0}' is not a valid value for the '{1}' option. The value must be of the form '{2}'.</value>
   </data>
+  <data name="IDS_FeatureFunctionPointers" xml:space="preserve">
+    <value>function pointers</value>
+  </data>
+  <data name="ERR_InvalidFunctionPointerCallingConvention">
+    <value>'{0}' is not a valid calling convention for a function pointer. Valid conventions are 'cdecl', 'managed', 'unmanaged', 'thiscall', and 'stdcall'.</value>
+  </data>
+  <data name="ERR_InvalidFunctionPointerCallingConvention_Title">
```

Comment:
>ERR_InvalidFunctionPointerCallingConvention_Title [](start = 14, length = 49)

Is this needed for an error?

Reply:
It's not needed in this PR directly anymore, but the next step will be binding the type, at which point I'll need to just readd this. So I left it in.

---
In reply to: [349347669](https://github.com/dotnet/roslyn/pull/39879#discussion_r349347669) [](ancestors = 349347669)